backend/routers/followup.py

```
@@ -53,7 +53,7 @@ async def schedule_followup(
 
 @router.get("/my-followups")
 async def get_my_followups(
-    current_user: dict = Depends(get_current_user),
+    current_user: dict = Depends(get_admin_user),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)